### ISSUE TITLE
Can't use multiple instances in global mode

### ISSUE THREAD

**Issue by alitecsolucoes** [mazipan/vue-currency-filter#197]
in main.js

```
import VueCurrencyFilter from 'vue-currency-filter'

Vue.use(VueCurrencyFilter, 
  {
    symbol: 'R$',
    thousandsSeparator: '.',
    fractionCount: 2,
    fractionSeparator: ',',
    symbolPosition: 'front',
    symbolSpacing: true
  },
  {
    name: 'valor',
    symbol: '',
    thousandsSeparator: '',
    fractionCount: 2,
    fractionSeparator: ',',
    symbolPosition: 'front',
    symbolSpacing: false
  }
)

```

in views

```
              <v-row wrap class="mx-0">
                <v-text-field
                  class="mx-2"
                  :value="cli_financeiro.mensalidade | currency"
                  @input="value => (cli_financeiro.mensalidade = value)"
                  label="Mensalidade"
                ></v-text-field>
                <v-text-field
                  class="mx-2"
                  :value="cli_financeiro.desconto | valor"
                  @input="value => (cli_financeiro.desconto = value)"
                  label="Desconto"
                ></v-text-field>
              </v-row>

```

give me an error

`vue.runtime.esm.js?2b0e:619 [Vue warn]: Failed to resolve filter: valor`

and if I put the **[ ]** in

```
Vue.use(VueCurrencyFilter, [
  {
    symbol: 'R$',
    thousandsSeparator: '.',
    fractionCount: 2,
    fractionSeparator: ',',
    symbolPosition: 'front',
    symbolSpacing: true
  },
  {
    name: 'valor',
    symbol: 'R$',
    thousandsSeparator: '.',
    fractionCount: 2,
    fractionSeparator: ',',
    symbolPosition: 'front',
    symbolSpacing: true
  }
])

```
I give no errors at all but both filters stop works

**Comment by mazipan** [comment]
I haven't release for multiple instances.
The code is already in master but not release yet on npm.

**Comment by mazipan** [comment]
Released on 5.1.0

**Comment by alitecsolucoes** [comment]
> Released on 5.1.0

thanks man
